xml/favicons.xml

```diff
@@ -2410,4 +2410,13 @@
     <param pos="0" name="service.vendor" value="PaperCut"/>
   </fingerprint>
 
+  <fingerprint pattern="^57801ef5135f6a6e8ea69baef85f8607$">
+    <description>frp - fast reverse proxy</description>
+    <!-- favicon.ico from frp version 0.48.0 -->
+
+    <example>57801ef5135f6a6e8ea69baef85f8607</example>
```

Comment:
Looks good to me; I double checked historical assets too, but it looks like it's been this md5sum for [~5 years](https://github.com/fatedier/frp/blob/2225a1781fc2774517b5d164c98675f09ba156e3/assets/frpc/static/favicon.ico)